src/store/modules/delegates.js

```diff
@@ -28,8 +31,21 @@ export default {
       })
     }
   },
+
   getters: {
     delegates: state => state.delegates,
-    forged: state => state.forged
+    forged: state => state.forged,
+
+    byPublicKey: (state) => publicKey => {
+      return state.delegates.find(delegate => {
+        return delegate.publicKey === publicKey
+      }) || false
```

Comment:
I'd use `null` (or even `undefined`) instead of `false` to not give the impression that the result is a boolean.